autoload/ale/fixers/biome.vim

```diff
@@ -1,9 +1,15 @@
 function! ale#fixers#biome#Fix(buffer) abort
     let l:executable = ale#handlers#biome#GetExecutable(a:buffer)
     let l:options = ale#Var(a:buffer, 'biome_options')
+    let l:apply = '--apply'
+
+    let l:unsafe = ale#Var(a:buffer, 'biome_fixer_apply_unsafe')
+    if (l:unsafe is 1 || l:unsafe is v:true)
+        let l:apply = '--apply-unsafe'
+    endif
 
     return {
-    \   'command': ale#Escape(l:executable) . ' format'
+    \   'command': ale#Escape(l:executable) . ' check ' . l:apply
     \       . (!empty(l:options) ? ' ' . l:options : '')
```

Comment:
Since we want to run only the formater, maybe is good idea to also add `--linter-enabled=false` to the command so linter is not executed?

Reply:
From what I understand, everything is intended that the user specificed in his configuration file. Both formatting and linting can be set there.

Reply:
Yeah I would prefer to run the fixer for both the linter and the formatter at the same time. WIthout this change it will only run the formatter.

One could add `--linter-enabled=false` to the options if they only wanted to fix the formatting, but there's a bug in `g:ale_biome_options`. It will apply both to the fixer and the LSP, but they take pretty different arguments and biome will bail if you give invalid ones. I think those should be split out into `ale_biome_lsp_options` and `ale_biome_fixer_options` or something

Reply:
I see, agree that is better to have different options for fixer and linter. Can be handled on a separate PR.